h/static/scripts/directive/annotation.coffee

```diff
@@ -45,6 +45,7 @@ AnnotationController = [
     @hasDiff = false
     @showDiff = undefined
     @timestamp = null
+    @element_created_at = new Date().getTime()
```

Comment:
maybe uuid instead.

Can we hope for sub-millisecond linking of annotation directives? Maybe. Maybe not.